src/ssl.c

```diff
@@ -16609,12 +16609,45 @@ int wolfSSL_get_server_tmp_key(const WOLFSSL* ssl, WOLFSSL_EVP_PKEY** pkey)
 
 static int sanityCheckProtoVersion(WOLFSSL_CTX* ctx)
 {
-    if ((ctx->mask & WOLFSSL_OP_NO_SSLv3) &&
-            (ctx->mask & WOLFSSL_OP_NO_TLSv1) &&
-            (ctx->mask & WOLFSSL_OP_NO_TLSv1_1) &&
-            (ctx->mask & WOLFSSL_OP_NO_TLSv1_2) &&
-            (ctx->mask & WOLFSSL_OP_NO_TLSv1_3)) {
-        WOLFSSL_MSG("All TLS versions disabled");
+    int sanityConfirmed = 0;
+
+#ifndef NO_TLS
```

Comment:
Comment:
"Ensure context method supports minimum version by checking exclusions."
Or something like that.

If that is what is being attempted here then I would expect different code.

Reply:
`sanityCheckProtoVersion` checks `ctx->mask` to see if any compiled in protocol versions are enabled. I added a comment to this function explaining this.